DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/SyncNameProvider.cs

```diff
@@ -56,7 +56,7 @@ public class SyncNameProvider : ISyncNameProvider
             }
             set
             {
-                _contentType = value ?? throw new ArgumentNullException($"Null is not an acceptable value for {nameof(ContentType)}.");
+                _contentType = value ?? throw new ArgumentNullException("value");
```

Comment:
please change the exception parameter from "value" to nameof(value)

Reply:
If you make this change on this branch, you also need to make the change in dev as well. Or you can cherry pick the commit.

Reply:
Update code to throw custom exception: SetArgumentNullException